hack/verify-codegen.sh

```diff
@@ -21,15 +21,24 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-"$( dirname "${BASH_SOURCE[0]}")/update-codegen-clients.sh"
-if ! git diff --quiet --exit-code -- pkg/client; then
+if [[ "$(git status --porcelain=1 -- api config/crd/bases pkg/crd/files pkg/generated)" != "" ]]; then
+    echo "Sorry, some relevant files are not in the current git commit."
+    echo "Correct checking can not be done."
+    git status -- api config/crd/bases pkg/crd/files pkg/generated
+    exit 1
+fi
+
+pwd
+make all-generated
```

Comment:
This seems a little circular. This is supposed to be just verify not verify-and-build-if-missing

Reply:
This follows the established pattern from upstream.
I added the preflight check you see earlier in this script.
Having `make` do the derivations would make it impossible for this script to do the preflight check.